bcda/client/bluebutton.go

```diff
@@ -176,6 +178,34 @@ func (bbc *BlueButtonClient) GetCoverage(beneficiaryID, jobID, cmsID, since stri
 	return bbc.getBundleData(u, jobID, cmsID, nil)
 }
 
+func (bbc *BlueButtonClient) GetClaim(mbi, jobID, cmsID, since string, transactionTime time.Time) (*models.Bundle, error) {
```

Comment:
Ditto for the "IncludeTaxNumbers" and "excludeSAMHSA" header values, those will be completed when we implement DCGEO-130 on the BFD